.yarn/versions/bb53227a.yml

```diff
@@ -0,0 +1,24 @@
+releases:
+  "@yarnpkg/builder": patch
+  "@yarnpkg/eslint-config": patch
+  vscode-zipfs: patch
+
+declined:
+  - "@yarnpkg/plugin-compat"
+  - "@yarnpkg/plugin-constraints"
+  - "@yarnpkg/plugin-essentials"
+  - "@yarnpkg/plugin-exec"
+  - "@yarnpkg/plugin-file"
+  - "@yarnpkg/plugin-interactive-tools"
+  - "@yarnpkg/plugin-link"
+  - "@yarnpkg/plugin-npm"
+  - "@yarnpkg/plugin-stage"
+  - "@yarnpkg/plugin-typescript"
+  - "@yarnpkg/plugin-version"
+  - "@yarnpkg/plugin-workspace-tools"
+  - "@yarnpkg/cli"
+  - "@yarnpkg/core"
+  - "@yarnpkg/fslib"
+  - "@yarnpkg/libui"
+  - "@yarnpkg/pnp"
+  - "@yarnpkg/shell"
```

Comment:
I still think it'd be safer to mark them minor; the next one won't take long to land anyway, and it's not the kind of thing we'd cherry-pick in a patch release.

Reply:
Alright, do you only the directly affected packages or everything?